### PR TITLE
[Enhancement] support last_query_id function

### PR DESCRIPTION
**Pull request by Vallishp**
## Proposed changes

Issue Number: close #xxx

support display previous query id.

mysql> select LAST_QUERY_ID();
No connection. Trying to reconnect...
Connection id:    0
Current database: *** NONE ***

+-----------------+
| last_query_id() |
+-----------------+
| Not Available   |
+-----------------+
1 row in set (0.08 sec)

mysql> select LAST_QUERY_ID();
+-----------------------------------+
| last_query_id()                   |
+-----------------------------------+
| e133f2641e0a411c-b73f1f6c67bedcf9 |
+-----------------------------------+
1 row in set (0.01 sec)

mysql>

### CODE REVIEW

**Comment by Vallishp** [comment]
run buildall

File: fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisLexer.g4 [hunk 1]
```diff
@@ -322,6 +322,7 @@ KILL: 'KILL';
 LABEL: 'LABEL';
 LARGEINT: 'LARGEINT';
 LAST: 'LAST';
+LAST_QUERY_ID: 'LAST_QUERY_ID';
```
Comment (morrySnow) [inline fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisLexer.g4:325]: i think we should not support use this function without parentheses, so add this to keyword list is not necessary

File: fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java [hunk 2]
```diff
@@ -879,6 +883,10 @@ public TUniqueId queryId() {
         return queryId;
     }
 
+    public TUniqueId getLastQuerId() {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java:886]:
```suggestion
    public TUniqueId getLastQueryId() {
```

**Comment by Vallishp** [comment]
run buildall

File: fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisLexer.g4 (hunk 1 above)
Comment (Vallishp) [reply fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisLexer.g4]: Thanks. updated

**Review by yiguolei — DISMISSED** [review]

File: fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4
Comment (morrySnow) [inline fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4:1]: all changes this file are not necessary

**Comment by Vallishp** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by morrySnow — APPROVED** [review]

**Pull request merged**